demo/simple/move-first-child.html

```diff
@@ -0,0 +1,50 @@
+<head>
```

Comment:
Missing some things from the preamble. (Also I think the viewport contains some redundant stuff.)

Reply:
all fixed and consistent across files.